subprojects/logging/src/integTest/groovy/org/gradle/internal/logging/console/AbstractConsoleBuildPhaseFunctionalTest.groovy

```diff
@@ -408,7 +409,7 @@ abstract class AbstractConsoleBuildPhaseFunctionalTest extends AbstractConsoleGr
         def buildFinished = server.expectAndBlock('build-finished')
 
         when:
-        gradle = executer.withTasks(":util:resolve").start()
+        gradle = executer.withTasks(":util:resolve", "--info").start()
```

Comment:
Is this necessary?

Reply:
I'll remove that on `master`. This is a left-over from debugging the test.

Reply:
Fixed by https://github.com/gradle/gradle/commit/817be07f7eb3bdc2c1dde966f54b85694691d3be.